SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -183,30 +185,52 @@ void USpatialReceiver::OnAuthorityChange(Worker_AuthorityChangeOp& Op)
 
 void USpatialReceiver::HandlePlayerLifecycleAuthority(Worker_AuthorityChangeOp& Op, APlayerController* PlayerController)
```

Comment:
potential driveby - const ref also potentially pointer to const for the controller